lib/general.sh

```diff
@@ -1271,6 +1279,13 @@ download_and_verify()
 		display_alert "Timeout from $server" "retrying" "info"
 		server="https://mirrors.tuna.tsinghua.edu.cn/armbian-releases/"
 	fi
+	
+	# switch to another china mirror if US timeouts
```

Comment:
This part has to be done differently.

Reply:
I just add another mirror source while the tuna mirror not working correctly